<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>862</width>
    <height>599</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="btnHit">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>490</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Hit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnStand">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>490</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Stand</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDoubleD">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>490</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Double down</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSplit">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>490</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Split</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSurrender">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>490</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Surrender</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblCurrentBet">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>500</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0.0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>480</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Amount left to bet goes here</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>490</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>500</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Amount bet:</string>
    </property>
   </widget>
   <widget class="QLabel" name="imgLblCard1">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Card1</string>
    </property>
   </widget>
   <widget class="QLabel" name="imgLblCard2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>40</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Card2</string>
    </property>
   </widget>
   <widget class="QLabel" name="imgLblCard3">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>40</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Card3</string>
    </property>
   </widget>
   <widget class="QLabel" name="imgLblCard4">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>40</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Card4</string>
    </property>
   </widget>
   <widget class="QLabel" name="imgLblCard6">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>40</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Card6</string>
    </property>
   </widget>
   <widget class="QLabel" name="imgLblCard5">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>40</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Card5</string>
    </property>
   </widget>
   <widget class="QLabel" name="imgLblCard12">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>210</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Card12</string>
    </property>
   </widget>
   <widget class="QLabel" name="imgLblCard9">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>210</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Card9</string>
    </property>
   </widget>
   <widget class="QLabel" name="imgLblCard7">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>210</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Card7</string>
    </property>
   </widget>
   <widget class="QLabel" name="imgLblCard8">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>210</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Card8</string>
    </property>
   </widget>
   <widget class="QLabel" name="imgLblCard10">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>210</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Card10</string>
    </property>
   </widget>
   <widget class="QLabel" name="imgLblCard11">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>210</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Card11</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>862</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuRules">
    <property name="title">
     <string>Info</string>
    </property>
    <addaction name="actionRules"/>
   </widget>
   <widget class="QMenu" name="menuGame">
    <property name="title">
     <string>Game</string>
    </property>
    <addaction name="actionNew_Game"/>
   </widget>
   <addaction name="menuGame"/>
   <addaction name="menuRules"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionRules">
   <property name="text">
    <string>Rules</string>
   </property>
  </action>
  <action name="actionNew_Game">
   <property name="text">
    <string>New Game</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>doubleSpinBox</sender>
   <signal>valueChanged(double)</signal>
   <receiver>lblCurrentBet</receiver>
   <slot>setNum(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>562</x>
     <y>539</y>
    </hint>
    <hint type="destinationlabel">
     <x>790</x>
     <y>552</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
